README.md

```diff
@@ -139,6 +139,10 @@ The values returned by the hook can be passed on to your custom components as pr
 
 This is used to send a message to the chat API. Send `true` for the optional `isRetry` flag to if this is retrying a previously failed message. This allows the internal logic to correctly link the next successful answer to the failed query.
 
+##### resetConversation: `() => void;`
+
+This is used to reset the chat. Message history and topics discussed previosuly are restored to their initial state.
```

Comment:
Can we expand upon this, so people understand the behavioral impact of the action? For example:

> This is used to reset the conversational context of the chat. The message history will be cleared and the chatbot will "forget" everything that's been discussed.

Also I just saw a typo: "previosuly" -> "previously"

Reply:
https://github.com/vectara/react-chatbot/pull/30/commits/40410b89849f62c19110758c93389090db72db9e